modules/prediction/scenario/prioritization/obstacles_prioritizer.cc

```diff
@@ -33,6 +35,51 @@ using common::adapter::AdapterConfig;
 using common::math::Box2d;
 using common::math::Vec2d;
 
+namespace {
+
+bool IsLaneSequenceInReferenceLine(const LaneSequence& lane_sequence,
+    const ADCTrajectoryContainer* ego_trajectory_container) {
+  for (const auto& lane_segment : lane_sequence.lane_segment()) {
+    std::string lane_id = lane_segment.lane_id();
+    if (ego_trajectory_container->IsLaneIdInReferenceLine(lane_id)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+int NearestFrontObstacleIdOnLaneSequence(const LaneSequence& lane_sequence) {
+  int nearest_front_obstacle_id = -10;
```

Comment:
Why the default value is -10?

Reply:
Just want to distinguish other obstacles and ego vehicle

Reply:
It is better to use a flag. Hard coded magic number is not recommended.

Reply:
I don't this it is necessary to make a flag only for this case.

Reply:
I used inf instead of the magic number -10. Please checkout the latest commit.

Reply:
Okay.